js/landing.js

```diff
@@ -1,42 +1,44 @@
-$(document).ready(function() {
+(function() {
```

Comment:
I think the document.ready gives us what we need so this wrapper is not necessary here